### DISCUSSION TITLE
Adding Filter to Project View in Projects Beta is difficult

### DISCUSSION THREAD

**Discussion by michael-hawker** [community/community#18761] [Projects and Issues]
There's a lot of helpful filter modifiers like "label:" and "repo:" available, but as I try and type and get the auto-complete helper to discover them, it never seems to work correctly.

Especially after adding one of them.

I think this has to do with understanding how `,` works and having to redo the `label:` quantifier. It's fairly unclear when typing though that you get into this state where I think it thinks you're adding to the value vs. trying to add the next modifier.

**Comment by 2percentsilk** [comment]
Thanks for your feedback @michael-hawker! Can you tell me a little bit more about your scenario? Is auto-complete not showing up at all? What are the cases where it's not working correctly if this is intermittent behavior? Are you trying to filter to multiple labels or repos? Is this primarily on the Table view or Board view? Feel free to add screenshots if that's easier. 

Also adding @rileybroughten as the PM who will be looking into filter improvements for projects.

**Reply by michael-hawker** [reply]
@2percentsilk @rileybroughten thanks for responding! Here's a breakdown of what I went through and the confusion I faced when trying to figure out how this worked with the Table View:

So, it starts off ok, in this example, I started typing 'label:':

![image](https://user-images.githubusercontent.com/24302614/174893718-77f65c32-a6ce-4fc0-bfc3-b9c31784f122.png)

and then used auto-complete to fill in an existing label on my issues:

![image](https://user-images.githubusercontent.com/24302614/174893503-4c8fbd9d-beb8-4373-ba4e-70f958ba547e.png)

But if I immediately type `,` to add another label after, nothing happens...

![image](https://user-images.githubusercontent.com/24302614/174893780-1320f71c-1064-428f-bca1-b235322ee2b8.png)

Looks like it's blue colored like the rest of the text.

If I instead add a `space` or add a `space` after the comma, I can't add more labels either:

![image](https://user-images.githubusercontent.com/24302614/174894008-1d19974d-5c56-4e53-b7d1-9137e27633b4.png)

![image](https://user-images.githubusercontent.com/24302614/174894058-9202d427-1e2d-4410-af44-4238ff16f5f3.png)

What I'm supposed to know to do is always use a space and search for `label:` again as the field like so:

![image](https://user-images.githubusercontent.com/24302614/174894185-3ff1881d-f200-4610-85f4-bca19f991923.png)

But if I try doing this immediately after a `,` without using `space` then it doesn't work (which is what to me filing this issue and a lot of confusion until I realized that the system is waiting on space characters to work).

![image](https://user-images.githubusercontent.com/24302614/174894379-72be1c3b-fa65-4396-b77b-96d231ad3364.png)

(see how it's all blue above and thinks its text or something here too)

I need to use the 'magic' `,space` to get it to show me the options:

![image](https://user-images.githubusercontent.com/24302614/174894593-d1cc055f-8755-45d4-a70a-98bacab48ae0.png)

In combination with [trying to figure out more advanced logic](https://github.com/orgs/github-community/discussions/15655#discussioncomment-2966427) and what was supported or not there, it just made it hard to learn how to even just easily add multiple conditions to the filter.
